src/version/config/theme.config.js

```diff
@@ -1,5 +1,8 @@
 import { connect } from 'react-redux';
```

Comment:
I don't understand the location of this file. Any theme configuration should be either in the core, or in one of the apps, but I don't understand what this one configures.

Reply:
Removed